src/components/cards/BiobankReportCard.vue

```diff
@@ -1,210 +1,119 @@
-
-    <div class="card biobank-card">
-      <div class="card-header">
-        <div class="row">
-          <div class="col-md-10">
-            <h3>{{biobank.data.name}}</h3>
-
-            <small>
-              <dl class="row">
-                <dt class="col-sm-3"><b>Collection types:</b></dt>
-                <dd class="col-sm-9"><i>{{ collectionTypes }}</i></dd>
-                <dt class="col-sm-3"><b>Number of collections:</b></dt>
-                <dd class="col-sm-9"><i>{{ biobank.data.collections.length }}</i></dd>
-                <dt class="col-sm-3"><b>Number of samples:</b></dt>
-                <dd class="col-sm-9"><i>{{ numberOfSamples }}</i></dd>
-              </dl>
-            </small>
-
-
-            <p>{{biobank.data.description}}</p>
+  <div class="mg-biobank-card container">
+    <loading
+      :active="isLoading"
+      loader="dots"
+      color="var(--secondary)"
+      background-color="var(--light)"
+    ></loading>
+    <div class="container-fluid" v-if="biobank.data && !this.isLoading">
+      <div class="row">
+        <div class="col">
+          <!-- Back to previous page buttons -->
+          <button class="btn btn-link" @click="back">
+            <i class="fa fa-angle-left" aria-hidden="true"></i> Back
+          </button>
+        </div>
+      </div>
 
-            <br/>
+      <div class="row">
+        <div class="col">
+          <!-- Title -->
+          <report-title type="Biobank" :id="biobank.data.id" :name="biobank.data.name"></report-title>
 
+          <div class="container">
             <div class="row">
-              <div class="col-md-6">
-                <ul class="list-unstyled">
-                  <li><b>{{ biobank.data.juridical_person }}</b></li>
-                  <li v-if="biobank.data.contact.address">{{ biobank.data.contact.address }}</li>
-                  <li>{{ biobank.data.contact.zip }} {{ biobank.data.contact.city }}</li>
-                  <li>{{ biobank.data.country.name }}</li>
-                </ul>
+              <div class="col-8">
+                <!-- Description -->
+                <report-description :description="biobank.data.description" :maxLength="500"></report-description>
+
+                <!-- Collections-->
+                <h3>Collections</h3>
+                <div v-for="collection in collectionsData" v-if="!collection.parentCollection" :key="collection.id">
+                  <biobank-report-collection :collection="collection"></biobank-report-collection>
+                </div>
               </div>
-              <div class="col-md-4">
-                <ul class="list-unstyled">
-                  <li><b>{{ biobank.data.contact.first_name }} {{ biobank.data.contact.last_name }}</b></li>
-                  <li>Email: {{ biobank.data.contact.email }}</li>
-                  <li>Tel: {{ biobank.data.contact.phone }}</li>
-                </ul>
+
+              <!-- Right side card -->
+              <div class="col-4">
```

Comment:
make it col-md-4 or col-sm-4 or so to prevent it from stubbornly sticking to the right even when it no longer fits